app/assets/javascripts/react_on_rails.js

```diff
@@ -156,10 +100,59 @@
     return consoleReplay;
   };
 
-  function createReactElement(componentName, propsVarName, props, domId, trace, generatorFunction) {
+  function forEachComponent(fn) {
+    var els = document.getElementsByClassName('react-component');
```

Comment:
I'd suggest a longer name to avoid any chance of class conflicts:

`'js-react-on-rails-react-component'`